src/backend/Level0/CHIPBackendLevel0.cc

```diff
@@ -1358,8 +1368,16 @@ hipError_t CHIPQueueLevel0::getBackendHandles(uintptr_t *NativeInfo,
     return hipSuccess;
   }
 
+  // create a new synchronous cmd list
+  CHIPContextLevel0 *ChipCtxZe = (CHIPContextLevel0 *)ChipContext_;
+  auto Desc = ((CHIPDeviceLevel0 *)ChipDevice_)->getNextComputeQueueDesc();
+  Desc.flags = ZE_COMMAND_QUEUE_MODE_SYNCHRONOUS;
```

Comment:
It seems that `ZE_COMMAND_QUEUE_MODE_SYNCHRONOUS` is assigned to incorrect field. `ZE_COMMAND_QUEUE_MODE_SYNCHRONOUS` is a member of `ze_command_queue_mode_t` but it is assigned to a field of `ze_command_queue_flag_t`.

Reply:
good catch - thanks